javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,35 @@
+const jWidth = 5;
+const jDeep = 11;
+const jHeight = 8;
+const jVolumeInMeters = jWidth * jHeight * jDeep;
+const jGardenSizeInM2 = 70;
+const jHouseCost = 1000000;
+jHousePrice = jVolumeInMeters * 2.5 * 1000 + jGardenSizeInM2 * 300
+//console.log(' Julia house price ' + jHousePrice)
+let jHousey;
+if(jHouseCost >= jHousePrice){
+    jHousey = 'Julia is Paying more';
+}
```

Comment:
If Julia is paying just the right amount, won't the logged message be a bit confusing? Do you think this can be enhanced?

Reply:
yes i have made changes in conditions , if julia and peter paying same amount so message will be loged out.